src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -123,6 +126,36 @@ public void clickIndicateMissingResponses() {
         click(indicateMissingResponsesCheckbox);
     }
 
+    public void clickRemindAllButtonAndWaitForFormToLoad() {
+        click(remindAllButton);
+        waitForRemindModalPresence();
+        WebElement remindButton = browser.driver.findElement(By.className("remind-particular-button"));
+        waitForElementToBeClickable(remindButton);
+    }
+
+    private void waitForRemindModalPresence() {
+        By modalBackdrop = By.className("modal-backdrop");
+        waitForElementPresence(modalBackdrop);
+    }
```

Comment:
Maybe this method could be put in `AppPage` as a public method? Modals tend to be used across several pages, and this method seems general enough to be put there.